frontend/src/pages/VersionTree/index.tsx

```diff
@@ -61,28 +64,64 @@ export default class VersionTree extends React.Component<{}, any> {
   constructor(props) {
     super(props);
     this.state = {
-      gitTreeData: []
+      gitTreeData: [],
+      modelData: []
     };
   };
-  public componentDidMount() {}
+  public async componentDidMount() {
+  }
 
   public async generateGitTree(gitgraph) {
-    const res = await axios.get(config.modelURL);
-    res.data.reverse().sort((a, b) => new Date(b.create_time) - new Date(a.create_time))
-    this.setState({gitTreeData: generateGitData(res.data)})
+    let res = await axios.get(config.modelURL);
+    let modelList = res.data.reverse();
+    this.setState({
+      gitTreeData: generateGitData(modelList),
+      modelData: modelList.filter(model => ['PYTORCH', 'None'].indexOf(model.engine) >= 0)
+    })
     gitgraph.import(this.state.gitTreeData);
     console.log(this.state.gitTreeData)
   }
 
   public render() {
     return (
-      <Gitgraph
-        options={{ template: gitGraphOption }}
-      >
-        {(gitgraph) => {
-          gitgraph.import(mockData);
-        }}
-      </Gitgraph>
+      <Row>
+        <Col span={2}>
+          <Gitgraph
+            options={{ template: gitGraphOption }}
+          >
+            {async (gitgraph) => {
+              // use real data
+              await this.generateGitTree(gitgraph)
+            }}
+          </Gitgraph>
+        </Col>
+        <Col span={22}>
+          <List
+            size="large"
+            itemLayout="horizontal"
+            dataSource={this.state.modelData}
+            renderItem={model => (
+              <List.Item>
+                <Space size="large">
+                  <Button shape="round" size="large" style={{ width: 120 }}> {model.parent_model_id ? model.dataset : 'Origin'} </Button>
```

Comment:
You should use the Tag component for these columns.
The table (no table header) should be like this:

Distribution tags | Version tags | Creator | Create time | Actions
-------------------|---------------|----------|--------------|---------
`ImageNet`, `Original` | `v1`        | [Avatar] Shanshan | 2 hours ago | [Edit]()
`CIFAR10`             | `v1`               | [Avatar] Shanshan | 2 minutes ago | [Edit]()